python/docs/source/reference/pyspark.sql/spark_session.rst

```diff
@@ -0,0 +1,54 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+=============
+Spark Session
+=============
+.. currentmodule:: pyspark.sql
+
+.. autosummary::
+    :toctree: api/
+
+The entry point to programming Spark with the Dataset and DataFrame API.
+To create a Spark session, you should use ``SparkSession.builder`` attribute.
+See also :class:`SparkSession`.
+
+.. autosummary::
+    :toctree: api/
+
+    SparkSession
```

Comment:
Here it contains `SparkSession` class but `python/docs/source/reference/pyspark.sql/window.rst` doesn't have `Window` class.

Reply:
Thanks. I think we can just remove `SparkSession` here, it's duplicated with `SparkSession` under "Core Classes"